compiler/front/cli_reporter.nim

```diff
@@ -1600,7 +1600,16 @@ proc reportBody*(conf: ConfigRef, r: SemReport): string =
         "` can be defined only in the same module with its type (" & r.typ.render & ")"
 
     of rsemUnexpectedTypeBoundOpSignature:
-      result = "signature for '" & r.symstr & "' must be proc[T: object](x: var T)"
+      let value =
+        case r.symstr
```

Comment:
```suggestion
        case r.symstr.normalize
```

Do normalization to catch `=cOpY` and/or `=c_o_p_y`. You should probably add a test case for that as well.